### DISCUSSION TITLE
Help with Start DreamBooth error [accelerate: command not found]

### DISCUSSION THREAD

**Discussion by wrk-foundry** [TheLastBen/fast-stable-diffusion#299] [Q&A]
I'm using the New Fast Method. When I try to run the Start DreamBooth cell, it throws this error:

![accelerate error](https://user-images.githubusercontent.com/87151837/198901525-65abaac2-7c8e-4f18-97b0-f133aadea3c4.JPG)

I have no prior experience with Colab so I appreciate any guidance. Thanks!

**Comment by TheLastBen** [comment]
you need to run the cells from the top everytime you connect to google colab

**Reply by wrk-foundry** [reply]
Hi - I have, in order:
Mount [1]
Setting up the environment [10], [3]
Downloading the model [5]
DreamBooth [7]
Instance Images [8]
Training [11]

**Reply by TheLastBen** [reply]
yes, you shouldn't get that error then
